icu4c/source/i18n/unicode/unumberformatter.h

```diff
@@ -235,14 +235,16 @@ typedef enum UNumberUnitWidth {
      *
      * @stable ICU 60
      */
-            UNUM_UNIT_WIDTH_HIDDEN,
+            UNUM_UNIT_WIDTH_HIDDEN = 6,
 
```

Comment:
Needed explicit numeric values because this is preceded by enum values conditionalized on #ifndef U_HIDE_DRAFT_API.